test/ussd_nurse_rapidpro.test.js

```diff
@@ -249,14 +310,13 @@ describe('app', function() {
                         })
                         .run();
                     });
-                    
+
                 });
 
               }); //end of start NurseConnect
-                 
+
         }); //end of state_start
-    
-    
+
+
     });
 });
-        
```

Comment:
Always ensure new line at the end of the file (there's usually a setting in your editor that automatically sets this for you)

Reply:
noted, will do